docs/user-guide/mvd-configuration.md

```diff
@@ -416,31 +528,15 @@ The following steps assume you have installed a Zowe runtime instance (which inc
 
       Where `ZWESIS_MYSRV` is the unique name of the new ZSS.
 
-5. To start the new ZSS, in SDSF enter the following command:
+5. To start the new ZIS, in SDSF enter the following command:
 
    ```
     /S ZWESIS02
    ```
 
-6. Make sure that the TSO user ID that runs the first ZSS started task also runs the new ZSS started task. The default ID is ZWESVUSR.
-
-7. In the new ZSS `server.json` configuration file, add a `"privilegedServerName"` parameter and specify the new ZSS name, for example:
-
-    ```
-   "productDir":"../defaults",
-    // All paths relative to zlux-app-server/bin
-    // In real installations, these values will be configured during the install.
-   "productDir":"../defaults",
-   "siteDir":"../deploy/site",
-   "instanceDir":"../deploy/instance",
-   "groupsDir":"../deploy/instance/groups",
-   "usersDir":"../deploy/instance/users",
-   "pluginsDir":"../defaults/plugins",
-   "privilegedServerName":"ZWESIS_MYSRV",
-   "dataserviceAuthentication": { ... }
-   ```
+6. Make sure that the TSO user ID that runs the first ZIS started task also runs the new ZIS started task. The default ID is ZWESVUSR.
 
-    **Note:** The instance location of `server.json` is `$INSTANCE_DIR/workspace/app-server/serverConfig/server.json`, and the defaults are stored in `$ROOT_DIR/components/app-server/share/zlux-app-server/defaults/serverConfig/server.json`
+7. In the new instance configuration file, either instance.env or zowe.yaml, add a `ZOWE_ZSS_XMEM_SERVER_NAME` parameter and specify the new ZIS name, for example `ZOWE_ZSS_XMEM_SERVER_NAME="ZWESIS_MYSRV"`
```

Comment:
You should specify that in zowe.yaml this is components.zss.crossMemoryServerName I think